Assets/Mirror/Editor/Weaver/Processors/ServerRpcProcessor.cs

```diff
@@ -48,7 +53,7 @@ public static MethodDefinition GenerateStub(MethodDefinition md, CustomAttribute
 
             // write all the arguments that the user passed to the Cmd call
             if (!NetworkBehaviourProcessor.WriteArguments(worker, md, RemoteCallType.ServerRpc))
-                return null;
+                return cmd;
```

Comment:
Why?

Reply:
because right after this method, we call generateSkeleton and it expects a non null function.

Not to worry,  this still shows an error in the console.

Reply:
already merged in master